cdk/src/main/java/dev/stratospheric/todoapp/cdk/DatabaseApp.java

```diff
@@ -45,6 +45,13 @@ public static void main(final String[] args) {
       applicationEnvironment,
       new PostgresDatabase.DatabaseInputParameters());
 
+    new BreadcrumbsDynamoTable(
```

Comment:
I thought it makes sense to put this into our database app as this is also a database-related infrastructure resources.

Reply:
Both good points. I'm not sure what the exact best practices for this are for CDK, but maybe, something like a fundamental `InfrastructureApp` that deploys everything without which the app couldn't run at all (e.g., network) and feature-based stacks for everything else (similar to the feature module pattern: https://angular.io/guide/feature-modules ).

Alternatively, as you suggested, we could indeed group aspects or infrastructure components together that serve a similar purpose (which'd more akin to the folder-by-type pattern: https://softwareengineering.stackexchange.com/questions/338597/folder-by-type-or-folder-by-feature ). However, in this particular instance, PostgreSQL and DynamoDB don't really have a whole lot in common, other than that they're both used for storing some kind of data. `DataStorageApp` might be a better name than `DatabaseApp` if we decide to go down that route.